tests/SprykerSdkTest/Zed/SprykGui/codeception.yml

```diff
@@ -67,7 +67,10 @@ suites:
                 - \SprykerTest\Zed\EventDispatcher\Helper\EventDispatcherHelper
                 - \SprykerTest\Zed\Twig\Helper\TwigHelper:
                     projectTemplatePath:
-                        - /src/*/Zed/*/Presentation/
+                        - /src/*/Zed/*/Presentation
+                        - /vendor/spryker/*/src/Spryker/Zed/*/Presentation
```

Comment:
This should be the default in the actual twig release :) This is how it should be done properly.
Not the Bundles/ path leak